test/relations.integration.js

```diff
@@ -1652,7 +1652,7 @@ describe('relations - integration', function() {
           expect(res.body).to.be.an('object');
           expect(res.body.error).to.be.an('object');
           expect(res.body.error.name).to.equal('Error');
-          expect(res.body.error.statusCode).to.equal(500);
+          expect(res.statusCode).to.equal(500);
```

Comment:
Could you please check with @davidcheung whether this change is needed. I am surprised that `body.error.statusCode` is not available.